src/components/edit/HighlightableBlockStyle.ts

```diff
@@ -8,29 +8,56 @@ export const firstTokenClassName = "FirstLyricToken";
 export const chordTargetClassName = "ChordTarget";
 export const chordSymbolClassName = "ChordSymbol";
 
-const highlightedSpaceStyle = (theme: Theme): CSSProperties => ({
+const highlightedSpaceStyle = (theme: Theme) => ({
     backgroundColor: theme.palette.primary.main,
     borderRadius: "0.3em",
 });
 
-const highlightedWordStyle = (theme: Theme): CSSProperties => ({
+const highlightedWordStyle = (theme: Theme) => ({
     color: theme.palette.primary.main,
 });
 
-const outlineStyle = (theme: Theme): CSSProperties => ({
+const outlineStyle = (theme: Theme) => ({
     borderStyle: "solid",
     borderColor: theme.palette.primary.main,
     borderRadius: "0.3em",
     borderWidth: "0.075em",
 });
 
 export interface HighlightChordLyricStyleOptions {
-    outline?: (theme: Theme) => CSSProperties;
+    dragOverOutline?: (theme: Theme) => {
+        color?: string;
+        borderColor?: string;
+    };
+    hoverOutline?: (theme: Theme) => { color?: string; borderColor?: string };
     customLyricClassSelector?: string;
     customChordTargetClassSelector?: string;
     customChordSymbolClassSelector?: string;
 }
 
+export interface HighlightableBlockStyles {
+    dragOver: string;
+    hoverable: string;
+}
+
+export const makeHighlightableBlockStyles = (
```

Comment:
one of the bigger refactors here

moving from makeStyles to cx(css()) which seems to be the way to generate class names for specific styles. 

seems like it can still be made in a very makeStyles paradigm - create the style object and then return a function that then calls useTheme and pass it into the style object, and create the className there